apps/ledger-live-mobile/src/components/Nft/NftListItem.tsx

```diff
@@ -158,6 +186,9 @@ const NftListItem = ({ nft, onPress }: Props) => {
       collectionStatus={collectionStatus}
       collectionMetadata={collectionMetadata}
       onPress={handlePress}
+      onLongPress={handleLongPress}
```

Comment:
here I think you don't need an intermediary handleLongPress (that does nothing more than calling onLongPress), you can directly pass `onLongPress`